openapi/openapi.yaml

```diff
@@ -445,6 +637,71 @@ components:
             $ref: '#/components/schemas/Namespace'
 
   responses:
+    Collections:
+      description: 'Response containing a page of Collections'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/CollectionsPage'
+
+    Collection:
+      description: 'Response containing a Collection'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Collection'
+
+    CollectionVersion:
+      description: 'Response contain a CollectionVersion'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/CollectionVersion'
+
+    CollectionVersions:
+      description: 'Response containing a page of CollectionVersions'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/CollectionVersionsPage'
+
+    CollectionVersionArtifact:
+      description: 'Response contain a CollectionVersionArtifact'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/CollectionVersionArtifact'
+
+    CollectionVersionArtifactDownload:
+      description: 'The collection artifact file binary contents'
+      content:
+        application/octet-stream:
+          schema:
+            title: 'CollectionVersionArtifactDownload'
+            description: 'The collection artifact file binary contents'
+            type: string
+            format: binary
+
+    CollectionImportAccepted:
+      description: 'Result of an accepted collection import. Includes the url of the import task'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/CollectionCreationResult'
+
+    CollectionUpdateAccepted:
+      description: 'Result of an accepted collection update.'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Collection'
```

Comment:
Not sure what this should be... convention would be it returns a Collection object, but if it really triggers an 'import' it may need to return a CollectionImportAccepted